content/en/guides/get-started/commands.md

```diff
@@ -0,0 +1,316 @@
+
+## Production Deployment
+
+Nuxt.js lets you choose between three modes to deploy your application: SSR, Static Generated, or SPA.
+
+### Server-Side Rendered Deployment (Universal SSR)
+
+To deploy a SSR application we use `target: server`, where server is the default value.
+
+<code-group>
+  <code-block label="npx" active>
+
+```bash
+nuxt build
+```
+
+  </code-block>
+  <code-block label="Yarn">
+
+```bash
+yarn build
+```
+
+  </code-block>
+  <code-block label="NPM">
+
+```bash
+npm run build
+```
+
+  </code-block>
+</code-group>
+
+Once you application is built you can use the `start` command to see a production version of your application
+
+<code-group>
+  <code-block label="npx" active>
+
+```bash
+nuxt start
+```
+
+  </code-block>
+  <code-block label="Yarn">
+
+```bash
+yarn start
+```
+
+  </code-block>
+  <code-block label="NPM">
+
+```bash
+npm run start
+```
+
+  </code-block>
+</code-group>
+
+<base-alert type="info">
+
+we recommend putting `.nuxt` in `.npmignore` or `.gitignore`.
+
+</base-alert>
+
+### Static Generated Deployment (Pre-rendered)
+
+Nuxt.js gives you the ability to host your web application on any static hosting.
+
+To deploy a static generated site make sure you have `target: static` in your `nuxt.config.js`.(For Nuxt >= 2.13:)
+
+```js{}[nuxt.config.js]
+export default {
+  target: 'static'
+}
+```
+
+<code-group>
+  <code-block label="npx" active>
+
+```bash
+nuxt generate
+```
+
+  </code-block>
+  <code-block label="Yarn">
+
+```bash
+yarn generate
+```
+
+  </code-block>
+  <code-block label="NPM">
+
+```bash
+npm run generate
+```
+
+  </code-block>
+</code-group>
+
+Nuxt.js will create a `dist` folder with everything inside ready to be deployed on a static hosting service.
+
+#### **Fail on Error**
+
+To return a non-zero status code when a page error is encountered and let the CI/CD fail the deployment or build, you can use the `--fail-on-error` argument.
+
+<code-group>
+<code-block label="npx" active>
+
+```bash
+nuxt generate --fail-on-error
+```
+
+  </code-block>
+  <code-block label="Yarn">
+
+```bash
+yarn generate --fail-on-error
+```
+
+  </code-block>
+  <code-block label="NPM">
+
+```bash
+npm run generate --fail-on-error
+```
+
+  </code-block>
+
+</code-group>
+
+As of Nuxt v2.13 there is a crawler installed that will now crawl your link tags and generate your routes when using the command `nuxt generate` based on those links.
+
+<base-alert>
+
+**Warning:** dynamic routes are ignored by the `generate` command when using Nuxt <= v2.12: [API Configuration generate](/api/configuration-generate#routes)
+
+</base-alert>
+
+<base-alert type="info">
+
+When generating your web application with `nuxt generate`, [the context](/guides/internals-glossary/context) given to [asyncData](/guide/async-data) and [fetch](/guide/vuex-store#the-fetch-method) will not have `req` and `res`.
+
+</base-alert>
+
+### Single Page Application Deployment (SPA)
```

Comment:
I think this section should be avoided. Nuxt.js apps are **ALWAYS** SPA. Regardless of `server` and `target` options.

Reply:
Fully agree, SPA wording should be removed out of the documentation as much as possible since it brings confusion.